### PR TITLE
Fix index out of range panic for reading opentelekomcloud_cts_tracker_v1

### LINKED ISSUE #517: [CTS] reading tracker can lead to panic
One of OTC customers faced panic during usage of `opentelekomcloud_cts_tracker_v1`

From customer debug log:
```log
2020-06-04T16:30:43.808+0200 [DEBUG] plugin.terraform-provider-opentelekomcloud_v1.17.1_x4: panic: runtime error: index out of range
2020-06-04T16:30:43.808+0200 [DEBUG] plugin.terraform-provider-opentelekomcloud_v1.17.1_x4:
2020-06-04T16:30:43.808+0200 [DEBUG] plugin.terraform-provider-opentelekomcloud_v1.17.1_x4: goroutine 1071 [running]:
2020-06-04T16:30:43.808+0200 [DEBUG] plugin.terraform-provider-opentelekomcloud_v1.17.1_x4: github.com/terraform-providers/terraform-provider-opentelekomcloud/opentelekomcloud.resourceCTSTrackerRead(0xc000366000, 0x21a3500, 0xc0006ee680, 0xc000366000, 0x0)
2020-06-04T16:30:43.808+0200 [DEBUG] plugin.terraform-provider-opentelekomcloud_v1.17.1_x4:     /opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/terraform-providers/terraform-provider-opentelekomcloud/opentelekomcloud/resource_opentelekomcloud_cts_tracker_v1.go:140 +0x978
2020-06-04T16:30:43.808+0200 [DEBUG] plugin.terraform-provider-opentelekomcloud_v1.17.1_x4: github.com/hashicorp/terraform-plugin-sdk/helper/schema.(*Resource).Refresh(0xc00052d480, 0xc000c56730, 0x21a3500, 0xc0006ee680, 0xc000825830, 0x10bec01, 0x1efe520)
```

### PR DESCRIPTION
**Pull request by outcatcher**
Fix #517

### CODE REVIEW

File: opentelekomcloud/resource_opentelekomcloud_cts_tracker_v1.go [hunk 1]
```diff
@@ -128,15 +126,15 @@ func resourceCTSTrackerRead(d *schema.ResourceData, meta interface{}) error {
 	}
 	trackers, err := tracker.List(ctsClient, listOpts)
 	if err != nil {
```
Comment (ShiChangkuo) [inline opentelekomcloud/resource_opentelekomcloud_cts_tracker_v1.go:128]: how  about put two conditions together?

Comment (outcatcher) [reply opentelekomcloud/resource_opentelekomcloud_cts_tracker_v1.go:128]: You mean, check for both 404 and empty trackers?

Comment (ShiChangkuo) [reply opentelekomcloud/resource_opentelekomcloud_cts_tracker_v1.go:128]: after reviewing the [golangsdk](https://github.com/huaweicloud/golangsdk/blob/master/openstack/cts/v1/tracker/requests.go#L21-L33), the 404 error happens rarely and your patch also includes the error, so feel free about my comment above.

Comment (outcatcher) [reply opentelekomcloud/resource_opentelekomcloud_cts_tracker_v1.go:128]: Just from my point of view, 404 for the tracker list should be impossible. So you're okay with merging PR as-is?

Comment (ShiChangkuo) [reply opentelekomcloud/resource_opentelekomcloud_cts_tracker_v1.go:128]: sure

**Pull request merged**